designs.js

```diff
@@ -30,17 +28,34 @@ function makeList(){
 
     const columnDelete = document.createElement('td');
 
-    const deleteButton = document.createElement('input')
+    const deleteButton = document.createElement('input');
     deleteButton.type = 'button';
     deleteButton.value = 'Delete';
     deleteButton.addEventListener('click', function(event) {
-      deleteRow(event)
+      deleteRow(event);
+    });
+
+    const upButton = document.createElement('input');
+    upButton.type = 'button';
+    upButton.value = 'Up';
+    upButton.addEventListener('click', function(event) {
+      upRow(event);
+    });
+
+    const downButton = document.createElement('input')
+    downButton.type = 'button';
+    downButton.value = 'Down';
+    downButton.addEventListener('click', function(event) {
+      downRow(event);
     });
 
     columnDelete.appendChild(deleteButton);
+    columnDelete.appendChild(upButton);
+    columnDelete.appendChild(downButton);
```

Comment:
Could you please rename columnDelete to columnButtons? Current name looks confusing.